src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -79,6 +79,15 @@ public ProcessResult updateCase(
                 EVENT_ID_ATTACH_SCANNED_DOCS_WITH_OCR
             );
 
+            log.info(
+                "Started event {} for updating case {} from exception record {}. Case type: {}. Case state: {}",
```

Comment:
maybe just stick to the format used for the second part of the log. No need to build a sentence imho

Reply:
What format string would you propose?

Reply:
Started CCD event. EventId: {}. Case id: {}. Exception record id: {}. Case type: {}. Case state: {}